### DISCUSSION TITLE
problem hidden value when a column is low

### DISCUSSION THREAD

**Discussion by linhnv-kozocom** [recharts/recharts#3648] [General]
I created a chart using Bar and add Labellist, I'm using `stackId`  for 2 columns. I met the problem when the value brown column is high and the yellow column is low then it's hidden my value
<img width="415" alt="Screen Shot 2023-07-07 at 10 04 07" src="https://github.com/recharts/recharts/assets/115438001/d5abbb4e-4692-4d45-a2c2-0eaa060185a0">
The designer give me a solution
<img width="438" alt="solution" src="https://github.com/recharts/recharts/assets/115438001/bf988cfb-a930-4b60-9da0-3fc0fc753fa4">
So I don't know how to overwrite a value in SVG tag

a demo for this problem 
https://codesandbox.io/s/stacked-bar-chart-forked-f2nhhz

Thank you for reading

**Comment by ckifer** [comment]
Hi! This is discussed a bit here if you'd like some more information about the issue https://github.com/recharts/recharts/issues/3492

I don't have a solution for you atm due to not being able to re-order on the z-axis. @Yilun-Sun might be able to help since he answered the linked issue

**Reply by linhnv-kozocom** [reply]
Thank you for your solution, I already tried but still can not, I changed Position and I move to decouple the LabelList rendering from the chart element, But It's not working :(

**Reply by linhnv-kozocom** [reply]
Thank you  I close this discussion, I used `reverseStackOrder` handle
